docs/parameters.yaml

```diff
@@ -64,14 +64,29 @@ components: ["origin", "registry", "director"]
 ---
 name: IssuerKey
 description: |+
+  [Deprecated] Use IssuerKeysDirectory instead. The key file set by this parameter will automatically move to IssuerKeysDirectory.
```

Comment:
I think this documentation is currently wrong, no?  That's the prior behavior of the patch series.